src/main/java/com/coinbase/prime/model/assets/Asset.java

```diff
@@ -27,6 +28,8 @@ public class Asset {
     private boolean tradingSupported;
     @JsonProperty("explorer_url")
     private String explorerUrl;
+    @JsonProperty("networks")
```

Comment:
You don't need this. the @JsonProperty decorator is only for converting snake_case to camelCase